MAIN/explainer3.py

```diff
@@ -0,0 +1,76 @@
+import os
+import asyncio
+from datetime import datetime
+
+from PresentationParser import PresentationParser
+from GPTCommunicator import GPTCommunicator
+from SlideExplanation import SlideExplanation
+from PresentationExporter import PresentationExporter
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from DB.database import Upload  # Import the User and Upload classes from your database module
+
+# Load environment variables from .env file
+load_dotenv()

```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/